pkg/runtime/controller.go

```diff
@@ -75,13 +74,21 @@ type Controller struct {
 }
 
 // RulesKind defines the config kind name of mixer rules.
-const RulesKind = "mixer-rules"
+const RulesKind = "mixer-rule"
```

Comment:
Nit: there's quite a few type and const declarations in this file, we should consider consolidating into `type` and `const` blocks.

Reply:
I define at the top if they are generally useful.
If they are not generally useful, I define them close to the point of use.

I have moved some consts around to that end now.